execution_engine/src/core/engine_state/engine_config.rs

```diff
@@ -28,6 +54,22 @@ pub struct EngineConfig {
     strict_argument_checking: bool,
     wasm_config: WasmConfig,
     system_config: SystemConfig,
+    /// A private network specifies a list of administrative accounts.
+    administrative_accounts: Arc<Mutex<Vec<AdministratorAccount>>>,
```

Comment:
Don't mutate `administrative_accounts` as the information is loaded from new chainspec before commit_upgrade is called